librz/include/rz_type.h

```diff
@@ -243,7 +251,8 @@ RZ_API bool rz_type_db_delete_base_type(RzTypeDB *typedb, RZ_NONNULL RzBaseType
 RZ_API RZ_OWN RzList /* RzBaseType */ *rz_type_db_get_base_types_of_kind(const RzTypeDB *typedb, RzBaseTypeKind kind);
 RZ_API RZ_OWN RzList /* RzBaseType */ *rz_type_db_get_base_types(const RzTypeDB *typedb);
 
-RZ_API RZ_OWN char *rz_type_db_base_type_as_string(const RzTypeDB *typedb, RZ_NONNULL const RzBaseType *type);
+RZ_API RZ_OWN char *rz_type_db_base_type_as_string(RZ_NONNULL const RzTypeDB *typedb, RZ_NONNULL const RzBaseType *type);
+RZ_API RZ_OWN char *rz_type_db_base_type_as_string_pretty(RZ_NONNULL const RzTypeDB *typedb, RZ_NONNULL const RzBaseType *type, RzTypePrintOpts opts);
```

Comment:
```suggestion
RZ_API RZ_OWN char *rz_type_db_base_type_as_string_pretty(RZ_NONNULL const RzTypeDB *typedb, RZ_NONNULL const RzBaseType *type, int opts);
```
This should be an int, because you could pass several values of `RzTypePrintOpts` as a single option.

Reply:
@DMaroo no need for `RZ_NONNULL` for `typedb` parameter. It's obvious that almost any function in the `rz_type_*` namespace expects nonnull type database, except the `*_new()` one.